catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -264,6 +264,21 @@
         <bundle>mvn:ddf.catalog.admin/catalog-admin-module-sources/${project.version}</bundle>
     </feature>
 
+    <feature name="catalog-registry-transformer" install="manual" version="${project.version}"
+             description="DDF Registry">
+        <bundle>mvn:ddf.catalog.registry/catalog-registry-cswinputtransformer/${project.version}</bundle>
+    </feature>
+
+    <feature name="catalog-registry-metacard-filter" install="manual" version="${project.version}"
+             description="DDF Registry">
+        <bundle>mvn:ddf.catalog.registry/catalog-registry-metacard-filter/${project.version}</bundle>
+    </feature>
+
+    <feature name="catalog-registry-policyplugin" install="manual" version="${project.version}"
+             description="DDF Registry Policy Plugin">
+        <bundle>mvn:ddf.catalog.registry/catalog-registry-policyplugin/${project.version}</bundle>
+    </feature>
+
```

Comment:
They don't strictly require each other but they should be run together. Moved them all into one feature.